javascript/organizing_your_javascript_code/project_restaurant_page.md

```diff
@@ -4,6 +4,15 @@ Let's use what we've learned and take a chance to continue practicing DOM manipu
 
 **Note: DOM elements should be created using JavaScript but styling can be done in a separate CSS file.**
 
+<div class="lesson-note lesson-note--tip" markdown="1">
+
+#### .gitignore
+
+When working with packages that are installed with npm, you don't need to track the contents of `node_modules` with git, or push those files to GitHub either. This is because the `package.json` file contains all the information, so that anyone can clone your project and install them on their machine with `npm install`.
```

Comment:
```suggestion
When working with packages that are installed with npm, you don't need to track the contents of `node_modules` with git, nor push those files to GitHub. This is because the `package.json` file contains all the information, so that anyone can clone your project and install them on their machine with `npm install`.
```
Tiniest of nits.